codersdk/deployment.go

```diff
@@ -1550,6 +1551,15 @@ Write out the current server config as YAML to stdout.`,
 			Value:  &c.GitAuthProviders,
 			Hidden: true,
 		},
+		{
+			Name:        "Git Auth Built-In GitHub",
+			Description: "Turns on/off the built-in GitHub App used for authenticating users with private repositories. Enabled by default.",
```

Comment:
```suggestion
			Description: "Toggles the built-in GitHub App used for authenticating users with private repositories..",
```
Everywhere we show the description we should already show the default value.